src/cheri/insns/jal_32bit.adoc

```diff
@@ -18,17 +18,15 @@ Jump to the target <<pcc>>.
 +
 Increment <<pcc>>.address by the sign-extended offset to form the target <<pcc>>.
 The <<pcc>> of the next instruction is sealed and written to `cd`.
++
+Both address increments use <<SCADDR>> semantics to determine the result.
+
+NOTE: A future extension may raise an exception on the branch instruction itself if a minimum sized instruction at the target <<pcc>> will raise a CHERI bounds exception.
 
```

Comment:
```suggestion
  Peforming the <<pcc>> bounds check at the branch source instead of on instruction fetch is helpful for debugging and can simplify the implementation of CPUs with very short pipelines.
  
```

@davidchisnall Maybe something like this as a note until we have an actual extension specified?